CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Fixed potential crash in SPIN when prompts are longer than encoder_seq_len - generation.max_length
 - Fixed crash when calling the `generate()` method of an SFT model with pipeline parallelism greater than two
 - Fixed crash when calling the `generate()` method of an SFT model with `compute_logprob=True` and string inputs
+- Fixed crash when mbs > 1 in DPO
```

Comment:
Do you mind just spelling it out as I'm not sure everyone is familiar with our acronyms?
```suggestion
- Fixed crash when `model.micro_batch_size` > 1 in DPO
```